cloud-gcp/src/main/java/com/sequenceiq/cloudbreak/cloud/gcp/loadbalancer/GcpLoadBalancerScheme.java

```diff
@@ -32,4 +33,20 @@ public static GcpLoadBalancerScheme getScheme(CloudLoadBalancer cloudLoadBalance
                 return INTERNAL;
         }
     }
+
+    public static GcpLoadBalancerScheme getScheme(String gcpType) {
```

Comment:
Could me move the conversion logic to the `LoadBalancerTypeConverter` class? This is getting weirdly busy for an enum class.

Reply:
Done, I think I've used a pattern where Enums can convert themselves, but cloudbreak seems to like more converters than i've seen before.

Reply:
They do love their converters. When I suggested this, I thought that `LoadBalancerTypeConverter` was a generic converter, and you could just add your logic to it. I now see the class I was thinking of is in the `cloud-aws-common` module, so not a shared resource. So that was my bad, but I do think having a separate converter does match the CB model better.